doc/sphinx/user/run-aspect/overview.md

```diff
@@ -141,7 +140,8 @@ total 932
 -rw-rw-r-- 1 bangerth bangerth 577138 Dec 11 10:07 parameters.tex
 drwxr-xr-x 2 bangerth bangerth   4096 Dec 11 10:08 solution
 -rw-rw-r-- 1 bangerth bangerth    484 Dec 11 10:08 solution.pvd
--rw-rw-r-- 1 bangerth bangerth    451 Dec 11 10:08 solution.visit
+-rw-rw-r-- 1 bangerth bangerth    451 Dec 11 10:08 solution.
```

Comment:
I don't think this change is correct.

Reply:
I stared at this for a good minute trying to figure out what I had done wrong, seeing my name all over that text but not recalling having anything to do with this PR. Then I realized that I had nothing to do with what is discussed here except that I'm being used as an example :-)